llvm/include/llvm/IR/PassManager.h

```diff
@@ -456,13 +461,6 @@ template <typename IRUnitT, typename... ExtraArgTs> class AnalysisManager {
   /// argument will not be called and this function will return false.
   /// Otherwise, we register the analysis returned by calling \c PassBuilder(),
   /// and this function returns true.
-  ///
-  /// (Note: Although the return value of this function indicates whether or not
-  /// an analysis was previously registered, there intentionally isn't a way to
-  /// query this directly.  Instead, you should just register all the analyses
```

Comment:
please keep the bit about just registering analyses without checking if they're already registered

Reply:
Something like "Although the return value of this function indicates whether or not
  /// an analysis was previously registered, you should just register all the analyses..."